ocs_ci/ocs/dr/dr_workload.py

```diff
@@ -31,19 +35,32 @@ def verify_workload_deployment(self):
 
 class BusyBox(DRWorkload):
     """
-    Class handlig everything related to busybox workload
+    Class handling everything related to busybox workload
 
     """
 
     def __init__(self, **kwargs):
-        workload_repo_url = kwargs.get("dr_workload_repo_url")
-        super().__init__("busybox", workload_repo_url)
+        workload_repo_url = config.ENV_DATA["dr_workload_repo_url"]
+        workload_repo_branch = config.ENV_DATA["dr_workload_repo_branch"]
+        super().__init__("busybox", workload_repo_url, workload_repo_branch)
+
+        self.workload_namespace = None
+        self.workload_pod_count = config.ENV_DATA["dr_workload_pod_count"]
+        self.workload_pvc_count = config.ENV_DATA["dr_workload_pvc_count"]
+
         # Name of the preferred primary cluster
-        self.preferred_primary_cluster = kwargs.get("preferred_primary_cluster")
-        self.target_clone_dir = kwargs.get("target_clone_dir")
+        self.preferred_primary_cluster = config.ENV_DATA.get(
+            "preferred_primary_cluster"
+        ) or (get_primary_cluster_config().ENV_DATA["cluster_name"])
```

Comment:
Is `preffered_primary_cluster` a preparation key for future scenarios where you'd specify the `preffered_primary_cluster` in your confs? Because I don't see this conf in the added `.yaml` to this PR.

Reply:
`preferred_primary_cluster` can be specified from `dr_workload.yaml` but it's not mandatory as we will fetch the value automatically if not provided in yaml.
I have updated the `dr_workload.yaml` in the PR and added a placeholder.